### PR TITLE
docs: Improved docs and links

### PR DESCRIPTION
**Pull request by okineadev**
I've made a few improvements, here's a list of them:

- Aligned the logo in `README.md` (https://github.com/wxt-dev/wxt/commit/46ee018be541723d47d1c3431c5787b3c07f25e6)
- Fixed `alt` attributes (https://github.com/wxt-dev/wxt/commit/725cf1ab5c20e94f072cfa90487af74073fd2b62)
- Fixed links (https://github.com/wxt-dev/wxt/commit/20ca5d770106876a06cb5674df557556c0cf4bc9, https://github.com/wxt-dev/wxt/commit/a02b0e38e64a1022e411176e4853cf525404ae7d)
- Symlinked readme from `README.md` to `packages/wxt/README.md` (https://github.com/wxt-dev/wxt/pull/970/commits/6318582e66da2132eb7532a536a9b878d23cd813)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,16 @@
 <h1 align="center">
-  <img style="vertical-align:middle" width="44" src="https://raw.githubusercontent.com/wxt-dev/wxt/HEAD/docs/public/hero-logo.svg" alt="WXT Logo">
+  <img align="top" width="44" src="https://raw.githubusercontent.com/wxt-dev/wxt/HEAD/docs/public/hero-logo.svg" alt="WXT Logo">
```
Comment (aklinker1) [inline README.md:2]: Haha thanks I've wanted to fix that for a long time! I thought I tried `vertical-align`, but maybe I tried using it as an attribute? Cool that the style works here.

File: packages/wxt/README.md [hunk 2]
```diff
@@ -1,39 +1,39 @@
...
   <span> </span>
-  <a href="https://www.npmjs.com/package/wxt" target="_blank"><img alt="npm" src="https://img.shields.io/npm/dm/wxt?labelColor=black&color=%234fa048"></a>
+  <a href="https://www.npmjs.com/package/wxt" target="_blank"><img alt="downloads" src="https://img.shields.io/npm/dm/wxt?labelColor=black&color=%234fa048"></a>
   <span> </span>
-  <a href="https://github.com/wxt-dev/wxt/blob/main/LICENSE" target="_blank"><img alt="NPM" src="https://img.shields.io/npm/l/wxt?labelColor=black&color=%234fa048"></a>
+  <a href="https://github.com/wxt-dev/wxt/blob/main/LICENSE" target="_blank"><img alt="license | MIT" src="https://img.shields.io/npm/l/wxt?labelColor=black&color=%234fa048"></a>
   <span> </span>
-  <a href="https://codecov.io/github/wxt-dev/wxt" target="_blank"><img alt="Codecov" src="https://img.shields.io/codecov/c/github/wxt-dev/wxt?labelColor=black&color=%234fa048"></a>
+  <a href="https://codecov.io/github/wxt-dev/wxt" target="_blank"><img alt="overage" src="https://img.shields.io/codecov/c/github/wxt-dev/wxt?labelColor=black&color=%234fa048"></a>
```
Comment (aklinker1) [inline packages/wxt/README.md:13]:
```suggestion
  <a href="https://codecov.io/github/wxt-dev/wxt" target="_blank"><img alt="Coverage" src="https://img.shields.io/codecov/c/github/wxt-dev/wxt?labelColor=black&color=%234fa048"></a>
```

File: packages/wxt/README.md [hunk 3]
```diff
@@ -1,39 +1,39 @@
...
-  <a href="https://wxt.dev/examples.html" target="_blank">Examples</a>
+  <a href="https://wxt.dev/examples" target="_blank">Examples</a>
   &bull;
   <a href="https://github.com/wxt-dev/wxt/blob/main/packages/wxt/CHANGELOG.md" target="_blank">Changelog</a>
   &bull;
   <a href="https://discord.gg/ZFsZqGery9" target="_blank">Discord</a>
 </p>
 
-![Example CLI Output](https://raw.githubusercontent.com/wxt-dev/wxt/HEAD/docs/assets/cli-output.png)
+![Example CLI Output](docs/assets/cli-output.png)
```
Comment (aklinker1) [inline packages/wxt/README.md:36]: Need to use full URLs so the assets show up correctly on NPM.
```suggestion
![Example CLI Output](https://raw.githubusercontent.com/wxt-dev/wxt/HEAD/docs/assets/cli-output.png)
```

File: README.md [hunk 4]
```diff
@@ -1,16 +1,16 @@
...
   <span> </span>
-  <a href="https://www.npmjs.com/package/wxt" target="_blank"><img alt="npm" src="https://img.shields.io/npm/dm/wxt?labelColor=black&color=%234fa048"></a>
+  <a href="https://www.npmjs.com/package/wxt" target="_blank"><img alt="downloads" src="https://img.shields.io/npm/dm/wxt?labelColor=black&color=%234fa048"></a>
   <span> </span>
-  <a href="https://github.com/wxt-dev/wxt/blob/main/LICENSE" target="_blank"><img alt="NPM" src="https://img.shields.io/npm/l/wxt?labelColor=black&color=%234fa048"></a>
+  <a href="https://github.com/wxt-dev/wxt/blob/main/LICENSE" target="_blank"><img alt="license | MIT" src="https://img.shields.io/npm/l/wxt?labelColor=black&color=%234fa048"></a>
   <span> </span>
-  <a href="https://codecov.io/github/wxt-dev/wxt" target="_blank"><img alt="Codecov" src="https://img.shields.io/codecov/c/github/wxt-dev/wxt?labelColor=black&color=%234fa048"></a>
+  <a href="https://codecov.io/github/wxt-dev/wxt" target="_blank"><img alt="overage" src="https://img.shields.io/codecov/c/github/wxt-dev/wxt?labelColor=black&color=%234fa048"></a>
```
Comment (aklinker1) [inline README.md:13]:
```suggestion
  <a href="https://codecov.io/github/wxt-dev/wxt" target="_blank"><img alt="Coverage" src="https://img.shields.io/codecov/c/github/wxt-dev/wxt?labelColor=black&color=%234fa048"></a>
```

File: packages/wxt/e2e/tests/__snapshots__/auto-imports.test.ts.snap
Comment (aklinker1) [inline packages/wxt/e2e/tests/__snapshots__/auto-imports.test.ts.snap:1]: Please don't modify the snapshot files, they're generated.

**Review by aklinker1 — CHANGES_REQUESTED** [review]
Thanks for the changes!

File: packages/wxt/README.md (hunk 2 above)
Comment (okineadev) [reply packages/wxt/README.md]: Done.

**Comment by okineadev** [comment]
Symbolic links do not work 100%

**Comment by okineadev** [comment]
This should be fixed

**Comment by okineadev** [comment]
Fixed.

Solution: https://mokacoding.com/blog/symliks-in-git/#:~:text=%23%20Good%20for%20Git%20repositories%0Acd%20./bar%20%26%26%20ln%20%2Ds%20../foo.md%20foo.md

File: packages/wxt/e2e/tests/__snapshots__/auto-imports.test.ts.snap
Comment (okineadev) [reply packages/wxt/e2e/tests/__snapshots__/auto-imports.test.ts.snap:1]: Done - https://github.com/wxt-dev/wxt/pull/970/commits/ba67a9bfb548254e58788773f5ebb317882d12e7

**Comment by aklinker1** [comment]
Note to self: double check the symlinked README gets packed in the non tarball correctly

**Comment by okineadev** [comment]
> Didn't revert the URL in the README: [#970 (comment)](https://github.com/wxt-dev/wxt/pull/970#discussion_r1746318890)

?

**Review by aklinker1 — COMMENTED** [review]
Here. We need to revert this change

File: README.md [hunk 5]
```diff
@@ -22,18 +22,18 @@
...
-  <a href="https://wxt.dev/examples.html" target="_blank">Examples</a>
+  <a href="https://wxt.dev/examples" target="_blank">Examples</a>
   &bull;
   <a href="https://github.com/wxt-dev/wxt/blob/main/packages/wxt/CHANGELOG.md" target="_blank">Changelog</a>
   &bull;
   <a href="https://discord.gg/ZFsZqGery9" target="_blank">Discord</a>
 </p>
 
-![Example CLI Output](https://raw.githubusercontent.com/wxt-dev/wxt/HEAD/docs/assets/cli-output.png)
+![Example CLI Output](docs/assets/cli-output.png)
```
Comment (aklinker1) [inline README.md:36]: Here, you didn't revert this change.

Comment (okineadev) [reply README.md]: Done

**Comment by okineadev** [comment]
@aklinker1

File: README.md [hunk 6]
```diff
@@ -22,11 +22,11 @@
 </p>
 
 <p align="center">
-  <a href="https://wxt.dev/guide/installation.html" target="_blank">Get Started</a>
+  <a href="https://wxt.dev/guide/installation" target="_blank">Get Started</a>
   &bull;
-  <a href="https://wxt.dev/api/config.html" target="_blank">Configuration</a>
+  <a href="https://wxt.dev/api/config" target="_blank">Configuration</a>
```
Comment (aklinker1) [inline README.md:27]: I don't think removing all HTML suffixes is meaningful. It breaks my redirects I have setup for old links too.

https://github.com/wxt-dev/wxt/blob/main/docs/public/_redirects

**Review by aklinker1 — CHANGES_REQUESTED** [review]
Published a test version to see what it looks like on NPM, and there's no README. Looks like `pnpm pack` ignores the symlinked README. So that solution won't work.

image

> https://www.npmjs.com/package/wxt/v/0.19.10-alpha1


Should we use [`automd`](https://automd.unjs.io/) for syncing the READMEs instead?

File: README.md (hunk 6 above)
Comment (okineadev) [reply README.md:27]: Look, in my opinion there should be a single standard - no suffix, because shorter links look better, about those links with a suffix - maybe it's good, but I don't see any point in it...

Comment (okineadev) [reply README.md:27]: I mean that if you went to a page with a suffix, you will go to a page with a suffix, if without a suffix, then without a suffix

I just want a single link type without a suffix to be used in the future

Comment (okineadev) [reply README.md:27]: I mean that if you went to a page with a suffix, you will go to a page with a suffix, if without a suffix, then without a suffix

I just want a single link type without a suffix to be used in the future

Comment (aklinker1) [reply README.md:27]: I agree, I'd prefer no suffix, but the current netlify and vitepress setup defaults to .html for everything. So that's what we should continue to use for now 🤷

**Comment by okineadev** [comment]
> Published a test version to see what it looks like on NPM, and there's no README. Looks like `pnpm pack` ignores the symlinked README. So that solution won't work.
> 
> image
> 
> > https://www.npmjs.com/package/wxt/v/0.19.10-alpha1
> 
> 
> Should we use [`automd`](https://automd.unjs.io/) for syncing the READMEs instead?

Later I will try to create a physical link (idk how it works, but it will probably be a more reliable link)

**Review by aklinker1 — APPROVED** [review]
